error: invalid memory address or nil pointer dereference

### Which version of kubefirst are you using?

2.7.4

### Which cloud provider?

k3d (local)

### Which DNS?

None specific

### Which installation type?

CLI

### Which distributed Git provider?

GitHub

### Did you use a fork of `gitops-template`?

No

### Which Operating System?

macOS

### What is the issue?

After a fresh installation when we run "kubefirst k3d root-credentials” we are geting below error

```
kubefirst k3d root-credentials
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x2 addr=0x20 pc=0x101837504]

goroutine 22 [running]:
github.com/konstructio/kubefirst-api/internal/vault.(*Configuration).GetUserPassword(0x140007f9ae0?, {0x140007f9ae0?, 0x1b}, {0x140000799a0, 0x1c}, {0x10232179a, 0x4}, {0x102378b3d, 0x10})
	github.com/konstructio/kubefirst-api@v0.115.0/internal/vault/users.go:20 +0x44
github.com/konstructio/kubefirst-api/pkg/credentials.ParseAuthData({0x10384d480, 0x140005c6a80}, {0x10231fe2e, 0x3}, {0x102364057, 0xd}, 0x14000083c84)
	github.com/konstructio/kubefirst-api@v0.115.0/pkg/credentials/root-credentials.go:77 +0x364
github.com/konstructio/kubefirst/cmd/k3d.getK3dRootCredentials(0x140004ee908, {0x10232172a?, 0x4?, 0x1023217e6?})
	github.com/konstructio/kubefirst/cmd/k3d/root-credentials.go:69 +0x448
github.com/spf13/cobra.(*Command).execute(0x140004ee908, {0x10549bf80, 0x0, 0x0})
	github.com/spf13/cobra@v1.7.0/command.go:940 +0x648
github.com/spf13/cobra.(*Command).ExecuteC(0x105420c80)
	github.com/spf13/cobra@v1.7.0/command.go:1068 +0x320
github.com/spf13/cobra.(*Command).Execute(...)
	github.com/spf13/cobra@v1.7.0/command.go:992
github.com/konstructio/kubefirst/cmd.Execute()
	github.com/konstructio/kubefirst/cmd/root.go:49 +0x3c
main.main.func2()
	github.com/konstructio/kubefirst/main.go:140 +0x1c
created by main.main in goroutine 1
	github.com/konstructio/kubefirst/main.go:139 +0xb9c
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

--- a/pkg/credentials/root-credentials.go
+++ b/pkg/credentials/root-credentials.go
@@ -73,7 +73,7 @@ func ParseAuthData(clientset kubernetes.Interface, cloudProvider string, domainN
 				fmt.Println("Cannot retrieve Vault token automatically. Please provide one here:")
 				fmt.Scanln(&vaultRootToken)
 			}
-			vault := vault.Configuration{}
+			vault := vault.Conf
 			kbotPassword, err = vault.GetUserPassword(
 				vaultURL,
 				vaultRootToken,

